mne/time_frequency/tests/test_spectrum.py

```diff
@@ -69,7 +70,26 @@ def test_compute_psd_errors(raw):
             True,
             False,
             "full",
+            "power",  # XXX: technically a default
         ],  # non-defaults
+        [
+            0,
+            np.inf,
+            None,
+            None,
+            None,
+            False,
+            256,
+            0,
+            None,
+            "mean",
+            "hamming",
+            None,
+            False,
+            True,
+            "length",
+            "complex",
+        ],  # complex  XXX: need to also test with non-defaults?
```

Comment:
> XXX: need to also test with non-defaults?

check test coverage, to see if any code paths are missed. If not, no need to test complex with non-default args.

Reply:
Yeah, coverage doesn't change so will fix comment.

Reply:
https://github.com/mne-tools/mne-python/blob/361db716396adfbe4ea0a2f9b837bc7ccd132dd4/mne/time_frequency/tests/test_spectrum.py#L92